specificatie/persoon.yaml

```diff
@@ -158,6 +174,18 @@ components:
           $ref: 'overlijden.yaml#/components/schemas/Overlijden'
         verblijfplaats:
           $ref: 'verblijfplaats.yaml#/components/schemas/AbstractVerblijfplaats'
+        immigratie:
+          $ref: 'immigratie.yaml#/components/schemas/Immigratie'
+        gemeenteVanInschrijving:
+          $ref: 'common.yaml#/components/schemas/Waardetabel'
+        datumAanvangAdreshouding:
+          $ref: 'datum.yaml#/components/schemas/GbaDatum'
+        datumAanvangAdresBuitenland:
+          $ref: 'datum.yaml#/components/schemas/GbaDatum'
+        datumInschrijvingInGemeente:
+          $ref: 'datum.yaml#/components/schemas/GbaDatum'
+        adressering:
+          $ref: '#components/schemas/Adressering'
```

Comment:
deze is niet correct, moet zijn $ref: 'adressering.yaml#components/schemas/Adressering'

Reply:
Aangepast